code/modules/wiremod/components/list/list_literal.dm

```diff
@@ -5,7 +5,7 @@
  */
 /obj/item/circuit_component/list_literal
 	display_name = "List Literal"
-	desc = "A component that returns the value of a list at a given index. Attack in hand to increase list size, right click to decrease list size."
+	desc = "A component that returns a list from whatever input you give it."
```

Comment:
i know it's proper wording for it, but for non-coders Returns can mean bad thing or something, maybe "creates a list"?

Reply:
```suggestion
	desc = "A component that creates a list from whatever input you give it."
```